mne/preprocessing/artifact_detection.py

```diff
@@ -4,12 +4,87 @@
 
 
 import numpy as np
-
 from ..annotations import (Annotations, _annotations_starts_stops)
 from ..transforms import (quat_to_rot, _average_quats, _angle_between_quats,
                           apply_trans, _quat_to_affine)
+from ..filter import filter_data
 from .. import Transform
-from ..utils import (_mask_to_onsets_offsets, logger)
+from ..utils import (_mask_to_onsets_offsets, logger, verbose)
+
+
+@verbose
+def annotate_muscle(raw, threshold=1.5, picks=None, min_length_good=.1,
```

Comment:
I would add `n_jobs=1` as a parameter and then pass `n_jobs` to `raw_copy.filter` and `raw_copy.apply_hilbert`